src/components/paragraph/stanford-card.tsx

```diff
@@ -53,13 +53,7 @@ const StanfordCard = ({
   }
 
   return (
-    <div
-      className={twMerge(
-        "relative",
-        !isHorizontal ? "centered mx-auto w-full lg:max-w-[980px]" : "p-0 md:rs-pt-5 lg:p-0"
-      )}
-      {...props}
-    >
+    <div className={twMerge("relative", !isHorizontal && "centered mx-auto w-full lg:max-w-[980px]")} {...props}>
```

Comment:
`clsx`... you know 😉

Reply:
I had an inkling that we'd want that. On it! 😆